understanding/20/audio-description-or-media-alternative-prerecorded.html

```diff
@@ -7,33 +7,29 @@
 </head>
 <body>
    <h1>Understanding Audio Description or Media Alternative (Prerecorded)</h1>
-   
+
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
          <dt>Goal</dt><dd>Prerecorded videos can be understood by more people.</dd>
          <dt>What to do</dt><dd>Provide a description of the visual content in videos.</dd>
          <dt>Why it's important</dt><dd>People who are blind or who cannot understand the visual content can have it described.</dd> 
       </dl>
-
    </section>
-   
+
    <section id="intent">
       <h2>Intent of Audio Description or Media Alternative (Prerecorded)</h2>
-      
-      
       <p>The intent of this Success Criterion is to provide people who are blind or visually
          impaired access to the visual information in a synchronized media presentation. This
          Success Criterion describes two approaches, either of which can be used.
       </p>
-      
       <p>One approach is to provide audio description of the video content. The audio description
          augments the audio portion of the presentation with the information needed when the
-         video portion is not available. During existing pauses in dialogue, audio description
-         provides information about actions, characters, scene changes, and on-screen text
-         that  are important and are not described or spoken in the main sound track.
+         video portion is not available. Audio description provides information about actions,
```

Comment:
I feel like even this inversion of the existing information is something we should be very cautious about.
In the existing text, the starting point is "During existing pauses in dialog...". It emphasizes that someone attempting to provide context for missing important visual information has to assess what openings/opportunities are available. This reflects the reality of how audio descriptions tend to be done, in my experience. The location and duration of pauses in the dialog is _the_ constraint under which AA level remediation (which tackles what can be done in synchronized video) takes place.

There is a slightly different feel when we invert the paragraph, IMO.